tap_zoho_crm/schemas/roles.json

```diff
@@ -1 +1,138 @@
-{}
\ No newline at end of file
+{
+    "type": "object",
+    "properties": {
+        "display_label": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "created_by__s": {
+            "type": [
+                "null",
+                "object"
+            ],
+            "properties": {
+                "id": {
+                    "type": [
+                        "null",
+                        "string"
+                    ]
+                },
+                "name": {
+                    "type": [
+                        "null",
+                        "string"
+                    ]
+                }
+            }
+        },
+        "modified_by__s": {
+            "type": [
+                "null",
+                "object"
+            ],
+            "properties": {
+                "id": {
+                    "type": [
+                        "null",
+                        "string"
+                    ]
+                },
+                "name": {
+                    "type": [
+                        "null",
+                        "string"
+                    ]
+                }
+            }
+        },
+        "forecast_manager": {
+            "type": [
+                "null",
+                "object"
+            ],
+            "properties": {
+                "id": {
+                    "type": [
+                        "null",
+                        "string"
+                    ]
+                },
+                "name": {
+                    "type": [
+                        "null",
+                        "string"
+                    ]
+                }
+            },
+            "additionalProperties": true
+        },
+        "share_with_peers": {
+            "type": [
+                "null",
+                "boolean"
+            ]
+        },
+        "api_name": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "modified_time__s": {
```

Comment:
Is this field really named liked this or you have flattened the schema on your end?!